### ISSUE TITLE
When use Image Target object that far from camera doesn't draw

### ISSUE THREAD

**Issue by bi92** [hiukim/mind-ar-js#71]
Hi 
I make image target to play video
everyThings work well but when camera go far ( even not far..) video and 3d object doesn't draw
![farclip_01](https://user-images.githubusercontent.com/68985482/141704190-656b5fa5-94cb-4327-9b7d-28cf149097ca.PNG)
 
![farclip_02](https://user-images.githubusercontent.com/68985482/141704194-257759c9-3d3e-4bb8-84e5-edf7d973e023.PNG)

when  the camera go close it works fine but the distance which can see is only 20cm...

**Comment by hiukim** [comment]
@bi92 Did you fix it?

**Comment by bi92** [comment]
Yes Because of size of image target.
when the image target size is too big , this things happened
I resized image to upder 1000 *1000 and it works well!
Thanks to make this great project : - )

**Comment by hiukim** [comment]
I see. but sounds like it's a problem with the library. I'll try to take a look later. 

If you happen to have a minimal and testable project that can produce this problem, please share.  thx.

**Comment by apps-estudioalfa** [comment]
Hi!

I have the same problem. I have generated the target on a paper, but i'm only able to scan it at a distance of max 60cm. If i'm further than that the objects and images do not appear (if i tilt the device at max distance of 60cm aprox. some objects are displayed or just a part of them, and so)

I have tried to scale the target to make it bigger or smaller but with no avail

thank you in advance.
